index.html

```diff
@@ -13,13 +13,14 @@
     <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
```

Comment:
move google font css to top of css calls, so the fonts load before you call them in your other CSS files